views/js/runner/plugins/content/accessibility/keyNavigation.js

```diff
@@ -150,33 +160,37 @@ define([
             var $itemElement = $(this);
             var id = 'item_element_navigation_group_'+itemNavigators.length;
             if($itemElement.hasClass('qti-interaction')){
-                $itemElement.off('.keyNavigation');
```

Comment:
that should not be removed because it enables overriding the key navigation from choice interaction's internal key navigation system